supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -162,6 +172,16 @@ When referring to the command, use pass:q[`g++`], marked up appropriately.
 
 *See also*:
 
+[[gluster-volume]]
+==== image:images/yes.png[yes] gluster volume (noun)
```

Comment:
Should "gluster be capitalized (i.e. "Gluster)?

Reply:
A quick search in docs shows it lowercase like this